### PR TITLE
Useless namespace

### PR DESCRIPTION
**Pull request by elbing**
Signed-off-by: Álvaro Jurado <elbingmiss@gmail.com>

### CODE REVIEW

File: sys/src/cmd/acid/builtin.c [hunk 1]
```diff
@@ -136,7 +136,6 @@ dosysr1(Node *r, Node*_)
 	r->op = OCONST;
 	r->type = TINT;
 	r->fmt = 'D';
-	fprint(2, "Thank Giacomo -- no sysr1 -- we'll bring it back\n");
```
Comment (rminnich) [inline sys/src/cmd/acid/builtin.c]: either verify that sysr1 is working again (it should be) or put an error and panic here. And, this should be a different PR.

Comment (rminnich) [inline sys/src/cmd/acid/builtin.c]: you can call sysr0 here, I brought it back a few months ago.

Comment (elbing) [inline sys/src/cmd/acid/builtin.c]: Restored.
